README.md

```diff
@@ -31,14 +31,23 @@ And get back a JSON response with either 'text' or 'error' top-level items set:
       }
     }
 
+### Languages
+
+#### Google
+
 Note that you should only set the `lang` parameter for languages that require it.
 The [documentation](https://cloud.google.com/vision/reference/rest/v1/images/annotate#imagecontext) informs us of the following:
 
 > In most cases, an empty value yields the best results since it enables automatic language detection.
 > For languages based on the Latin alphabet, setting languageHints is not needed.
 > In rare cases, when the language of the text in the image is known, setting a hint will help get better results
 > (although it will be a significant hindrance if the hint is wrong).
-> Text detection returns an error if one or more of the specified languages is not one of the supported languages.
+> Text detection returns an error if one or more of the specified languages is not
+> one of the [supported languages](https://cloud.google.com/vision/docs/languages).
```

Comment:
My tiny comment is that I didn't get any error in the UI when I specified an invalid language code.  Not sure if this patch is supposed to throw an error, but it doesn't matter much since we'll be adding language validation very soon.

Reply:
Oh interesting. You don't get any error when using `en` with Tesseract? But yeah, we can sort out all this when we add the validation.

Reply:
Using `en` with Tesseract I get an ErrorException: `Notice: fwrite(): write of 8192 bytes failed with errno=32 Broken pipe`

Reply:
Sorry for the confusion. I was trying to trigger an error using  Google since the reference is to the Google languages. 

For `Tesseract`, I'm also getting the Symfony Exception page with the message `Notice: fwrite(): write of 8192 bytes failed with errno=32 Broken pipe` when I submit an invalid language.